docs/source/installation.rst

```diff
@@ -97,14 +97,14 @@ Since Xvfb will require an X server (we also recommend to install XQuartz packag
 
 - First option::
 
-    $ export DISPLAY=:0
-    $ Xvfb :0 -screen 1920x1080x24 > /dev/null 2>1 &
-    $ pytest -svv fury
+    export DISPLAY=:0
+    Xvfb :0 -screen 1920x1080x24 > /dev/null 2>1 &
+    pytest -svv fury
 
 - Second option::
 
```

Comment:
Same as above and for the rest of the shell commands.

Reply:
I have done as you can review out my latest commit.